### ISSUE TITLE
There is no in situ surveyed shoreline data here. In the long-term, is this a mistake?

### ISSUE THREAD

**Issue by ddbuscombe** [kvos/SDS_Benchmark#13]
Hi @kvos 

I am about to embark on a new benchmark exercise with Coastseg/Zoo and would like to use the same 4 sites as here, as well as a number of additional sites (Madeira beach, Honolulu, Cala Millor, Slapton, Perranporth, and maybe others).

I was under the impression that the in situ survey data was worked up in a testable format that could be used in conjunction with these scripts. However, I'm not seeing any csv files here that contain the in situ shoreline data, i.e. chainage values from start of transect. 

I know you've made a great job at the preprocessing script, but it feels like a big duplication of effort, and I worry a lot about the links going dead in the future and maintaining access to these data

I am in the process of working up Slapton, Perranporth, Cala Millor, and other sites that have available survey data. I intend to make this available for everyone. Is this still a good location to post these files? Or should I start a new repo?

Any input would be greatly appreciated!  

Thanks, Dan

**Comment by kvos** [comment]
yes notebook 1_preprocess_datasets will prepare and standardise all the data for you. I initially added the in situ directly to the repo but got told to remove it and instead put instructions on how to download and prepare it so that I'm not bypassing the data providers.

**Comment by ddbuscombe** [comment]
Ok, thanks for responding. I think adding the data to the repo is a good idea and will do that when I have all the datasets (original 4, plus new ones). The data providers don't maintain their links.

**Comment by kvos** [comment]
hi @dbuscombe-usgs , please see this PR #11 from 5 months ago. This was a major bug picked up by @yongjingmao showing that the tide level time-series for Torrey Pines and Truc Vert are wrong in the repo (due to inverted X and Y coordinates between different versions of pyfes). We have replaced the tide time-series files with the correct ones but I still need to update the notebooks and eventually the figures in the paper. Keep this in mind when benchmarking CoastSeg, and also a good lesson learnt is to double-check the FES2014 tide time-series against in-situ when adding new sites to the benchmark.
